README.md

```diff
@@ -90,7 +90,7 @@ Here’s a quick command to evaluate using the Accelerate backend:
 ```shell
```

Comment:
All the tasks edit are removing the truncating few shots, since it's no longer used